documentation/src/main/asciidoc/cheat-sheets/cache-configuration.asciidoc

```diff
@@ -151,7 +151,8 @@ You should store indexes in JVM heap memory only for small datasets.
 </distributed-cache>
 ----
 
-Index reader:: The index reader provides access to the indexes to perform queries.
+==== Index reader
+The index reader provides access to the indexes to perform queries.
 As the index content changes, {brandname} needs to refresh the reader so that search results are up to date.
 +
```

Comment:
```suggestion
```
No need for + here

Reply:
I think that's just how the diff looks like. There is no + at the beginning of the line

Reply:
Sorry now I understand

Reply:
What it does is that it aligns the code block with the text above. I'd prefer keeping the + there so if the text is indented the config example would be as well.

Reply:
When I try to view it, `+`  appears 

![Screenshot from 2023-07-24 20-03-13](https://github.com/infinispan/infinispan/assets/97534722/28d88fc0-9c4a-4dfd-a0df-1764ca93d28a)

Reply:
You're right. I missed this. Now it's fixed